src/tools/FreehandSculpterMouseTool.js

```diff
@@ -520,10 +532,12 @@ export default class FreehandSculpterMouseTool extends BaseTool {
    * _checkSpacing - description
    *@modifies indiciesToInsertAfter
    *
-   * @param  {type} i                     The index to check.
-   * @param  {type} points                The points.
-   * @param  {type} indiciesToInsertAfter The working list of indicies to
-   *                                      insert new points after.
+   * @param {number} i - The index to check.
+   * @param {Object} points - The points.
+   * @param {Array} indiciesToInsertAfter - The working list of indicies to insert new points after.
+   * @param {number} maxSpacing
+   * @modifies indiciesToInsertAfter
```

Comment:
`@modifies` isn't part of the `JSDoc` tag list yet [docs](http://usejsdoc.org/). Its not being released until 3.6.0. I'm not saying we shouldn't preemptively tag stuff now, but it feels a bit odd to have a tag that doesn't do anything. What do you think @dannyrb @swederik ?

Reply:
Ah yeah, I might have gotten that from the closure compiler docs. I'll take it out.

Reply:
So I have removed that line, but it turns out this is in a bunch of places in this file but also `FreehandMouseTool.js` and `BaseBrushTool.js`.

Reply:
I'm not opposed to non-standard tags. `Docma`, what we use to generate API docs, supports non-standard tags, or we can simply strip them out when generating.

If it adds helpful information, and we use the tag consistently, then it feels like a win to me.

Reply:
@dannyrb lets just review if we can make Docma happy with this sometime in the future and then we'll add it back in if it is?

Reply:
I was responding more to @mikehazell's note about its usage in other places. No need to strip all of those out. If this has already been removed from this particular block, that's fine.

https://onury.io/docma/api#Docma~BuildConfiguration
`jsdoc.allowUnknownTags`